src/shared_protos/README.md

```diff
@@ -0,0 +1,3 @@
+# Updating
+
+To update this metadata folder, copy the .proto files from <https://github.com/ni/grpc-device/tree/main/source/protobuf>.
```

Comment:
Is "metadata folder" still an accurate way to refer to this?

Reply:
Changed it to say just "this folder".